requirements/requirements.txt

```diff
@@ -13,3 +14,4 @@ omegaconf
 optax
 protobuf~=3.20
 tensorboard_logger
+type_enforced
```

Comment:
Doesn't look like you use this
```suggestion
```

Reply:
it's imported in `gigastep_env.py` but they dont have it included in their requirements so removing it from here would make gigastep unusable

Reply:
Wow that's annoying, can you make a PR on their repo to add this

Reply:
Thinking about this can we add a comment here:  `# needed because gigastep is missing this dependency`